### ISSUE TITLE
[Question] Twitch chat integration

### ISSUE THREAD

**Issue by ddteam90** [iChun/ContentCreatorIntegration-IssuesAndDocumentation#23]
What  I need to do to execute the command in the game on the message in the twitch chat?

**Comment by iChun** [comment]
Can you rephrase that question?

**Comment by ddteam90** [comment]
For example:
Smb write in the twitch chat "!redeem chance"
And the game, after this message, gives me a Chance Cube?
How i can do this?

**Comment by iChun** [comment]
Sorry I didn't realise you had replied. Did you still need help with this?

**Comment by MeinKraft** [comment]
if you still need help.  for StreamElements, Create a type called "redemption". Add an Event. Set the condition to 'VariableCondition', set 'VariableName' to "redemption" and set 'VariableResult' to "chance" (or the name of item you want to !redeem)

Then add a 'CommandOutcome' to the 'outcomes'. Set to 'ExecuteAsSelf = True' and set 'Command' to something like "/setblock ~ ~ ~ chancecubes:chance_icosahedron 1 0"
